popup.css

```diff
@@ -0,0 +1,292 @@
+  letter-spacing: 2px;
+  padding: 1em 1.5em;
+  color: white;
+  border-radius: 3px;
+}
+
+.box-tags-popup {
+  display: flex;
+  justify-content: flex-start;
+  list-style: none;
+  padding: 0%;
+}
+
+.box-tags-popup li {
+  margin: 10px 10px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  list-style: none;
+  padding: 8px 12px;
+  background: #ebf0ee;
+  color: #3a4a42;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.75em;
+  line-height: 1em;
+  letter-spacing: 0.03em;
+  border-radius: 4px;
+}
+
+h2 {
+  display: flex;
+  justify-content: flex-start;
+  width: 100%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.modal-p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 30px;
+  color: #344563;
+}
+
+@media (min-width: 365px) {
+  .modal-image {
+    overflow: hidden;
+    box-sizing: border-box;
+    border-radius: 8px;
+    width: 100%;
+    max-height: 50vh;
+    object-fit: cover;
+  }
+
+  .box-tags-popup {
+    display: flex;
+    justify-content: center;
+    list-style: none;
+    padding: 0%;
+  }
+
+  .close {
+    background: #ebecf0;
+    mix-blend-mode: multiply;
+    padding: 6px 10px;
+    border-radius: 8px;
+    color: #aaa;
+    float: right;
+    font-size: 20px;
+    font-weight: bold;
+    margin-bottom: 6%;
+  }
+
+  .close:hover,
+  .close:focus {
+    color: black;
+    text-decoration: none;
+    cursor: pointer;
+  }
+
+  .modal-btn-template {
+    display: none;
+  }
+
+  .modal-btn-template-mob {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+  }
+
+  .modal-btn-template-mob a {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    align-items: center;
+    flex-basis: 40%; /* Adjust the value as needed */
+    margin-bottom: 10px; /* Adjust the spacing between buttons */
+    text-decoration: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .modal-btn-template-mob {
+    display: none;
+  }
+
+  .modal-btn {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .modal-btn-template {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    width: 100%;
+  }
+
+  .modal-btn-template a {
+    display: flex;
+    justify-content: flex-end;
+    width: 22%;
+    text-decoration: none;
+  }
+
+  .modal-see-live {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 1.5em;
+    margin-bottom: 1.5em;
+    margin-right: 1%;
+    border: none;
+    align-self: center;
+    background: var(--color-primary);
+    font-weight: 500;
+    font-size: 16px;
+    letter-spacing: 2px;
+    padding: 1em 1.5em;
+    color: white;
+    border-radius: 3px;
+  }
```

Comment:
- [ ] Please make sure you follow the given Figma design when implementing the button styling here as the width of the button is causing the text to break. please make sure to fix that. ✔
- Here's how it looks:
  ![image](https://github.com/Rifat8080/Portfolio-site/assets/104968236/8910f50f-86d9-4ab5-90dc-4077f13ebbe5)
- [ ] Also please make sure you implement the `hover` & `active` states of the button as shown in the below Figma design 👇
  ![firefox_kz6PEcoelc](https://github.com/Rifat8080/Portfolio-site/assets/104968236/fb1a7eb6-432a-49e2-acad-533d9c487736)